src/libraries/Common/tests/TestUtilities/System/AssertExtensions.cs

```diff
@@ -268,6 +268,27 @@ public static void ThrowsIf<T>(bool condition, Action action)
             }
         }
 
+        public static T ThrowsExceptionAssignableTo<T>(Action action)
```

Comment:
How does this differ from Assert.ThrowsAny?

Reply:
I just assumed that was the same as `AssertExtensions.ThrowsAny` and didn't look at its signature...